rootfs/Dockerfile

```diff
@@ -1,13 +1,20 @@
-FROM alpine:3.3
+FROM quay.io/deis/base:0.2.0
 
-# add logger user
-RUN addgroup -S stdout-metrics && \
-	adduser -S -G stdout-metrics -H -h /opt/stdout-metrics -D stdout-metrics
+RUN adduser --system \
+	--shell /bin/bash \
+	--disabled-password \
+	--home /opt/stdout-metrics \
+	--group \
+	stdout-metrics
 
-ADD . /
+COPY . /
```

Comment:
I'd like to avoid copying the entire `rootfs` directory, and just copying the binary (and whatever other artifacts are necessary at runtime).

However, that task could be for a different PR, so not blocking for this PR. I'll add issues to repositories this applies to